src/v/resource_mgmt/cpu_profiler.cc

```diff
@@ -178,6 +186,8 @@ cpu_profiler::override_and_get_results(
     // enable the profiler if disabled pre-override.
     on_enabled_change();
 
+    auto polling_start_time = ss::lowres_clock::now();
```

Comment:
I guess the doc for `cpu_profiler::override_and_get_results` should be updated now? 

Maybe the name could even simply get something like "collect_results_for_period" since what is really doing is collecting results starting _now_ and running through period. The override is just an internal detail of what needs to happen to make that work and indeed does not always occur (if the profiler was already running).

Reply:
The comments have been updated. The name definitely makes more sense. Switching to it now.